fleet_adapter_template/config.yaml

```diff
@@ -2,86 +2,81 @@
 # RMF Fleet parameters
 
 rmf_fleet:
-  name: "DeliveryBot"
+  name: "tinyRobot"
+  fleet_manager:
+    prefix: "http://127.0.0.1:7000"
+    user: "some_user"
+    password: "some_password"
   limits:
-    linear: [0.4, 0.2] # velocity, acceleration
-    angular: [0.3, 0.35] # velocity, acceleration
+    linear: [0.5, 0.75] # velocity, acceleration
+    angular: [0.6, 2.0] # velocity, acceleration
   profile: # Robot profile is modelled as a circle
-    footprint: 0.5 # radius in m
-    vicinity: 0.6 # radius in m
-  reversible: False # whether robots in this fleet can reverse
+    footprint: 0.3 # radius in m
+    vicinity: 0.5 # radius in m
+  reversible: True # whether robots in this fleet can reverse
   battery_system:
-    voltage: 24.0 # V
-    capacity: 40.0 # Ahr
-    charging_current: 26.4 # A
+    voltage: 12.0 # V
+    capacity: 24.0 # Ahr
+    charging_current: 5.0 # A
   mechanical_system:
-    mass: 80.0 # kg
-    moment_of_inertia: 20.0 #kgm^2
-    friction_coefficient: 0.20
+    mass: 20.0 # kg
+    moment_of_inertia: 10.0 #kgm^2
+    friction_coefficient: 0.22
   ambient_system:
     power: 20.0 # W
   tool_system:
-    power: 760.0 # W
-  recharge_threshold: 0.20 # Battery level below which robots in this fleet will not operate
+    power: 0.0 # W
+  recharge_threshold: 0.10 # Battery level below which robots in this fleet will not operate
   recharge_soc: 1.0 # Battery level to which robots in this fleet should be charged up to during recharging tasks
-  publish_fleet_state: True
+  publish_fleet_state: 10.0 # Publish frequency for fleet state, ensure that it is same as robot_state_update_frequency
   account_for_battery_drain: True
   task_capabilities: # Specify the types of RMF Tasks that robots in this fleet are capable of performing
     loop: True
-    delivery: False
+    delivery: True
     clean: False
     finishing_request: "nothing" # [park, charge, nothing]
 
-# DeliveryBot CONFIG =================================================================
+# TinyRobot CONFIG =================================================================
 
 robots:
   # Here the user is expected to append the configuration for each robot in the
   # fleet.
   # Configuration for first robot in this fleet
-  deliverybot1:
+  tinyRobot1:
     robot_config:
-      base_url: "http://192.168.0.1:8888" # API endpoint for the robot
-      user: "some_user"
-      password: "some_password"
-      # max_delay: 10.0 # allowed seconds of delay of the current itinerary before it gets interrupted and replanned
+      max_delay: 15.0 # allowed seconds of delay of the current itinerary before it gets interrupted and replanned
+      filter_waypoints: False
```

Comment:
The template does not have a `filter_waypoints()` function so let's remove this parameter